src/realtime_pollen_calibration/update_strength.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) 2022 MeteoSwiss, contributors listed in AUTHORS
+
+# Distributed under the terms of the BSD 3-Clause License.
+
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""A module for the update of the pollen emission strength."""
+
+# Standard library
+from datetime import datetime, timedelta
+
+# Third-party
+import numpy as np
+import xarray as xr
+from eccodes import (  # type: ignore
+    codes_get,
+    codes_get_array,
+    codes_grib_new_from_file,
+    codes_release,
+)
+
+# First-party
+from realtime_pollen_calibration import utils
+
+
+def read_pov_file(pov_infile, pol_fields, config_obj):
```

Comment:
There is also a function `read_pov_file` in the file `update_phenology.py`. It is hard to tell why they are different without a doc string. Maybe consider adding one to justify the duplication.

Reply:
The two functions are not doing the same. They could be unified, to be discussed. For now I added a docstring.